src/main/java/grok/Grok.java

```diff
@@ -0,0 +1,89 @@
+                System.out.println("Bye. Hope to see you again soon! Keep Grokking :)");
+                printLine();
+                break;
+            } else if (input.equals("list")) {
+                printLine();
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    System.out.println((i + 1) + ". " + tasks[i]);
+                }
+                printLine();
+            } else if (input.startsWith("mark ")) {
+                int taskNumber = Integer.parseInt(input.split(" ")[1]) - 1;
+                tasks[taskNumber].markAsDone();
+                printLine();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println(tasks[taskNumber]);
+                printLine();
+            } else if (input.startsWith("unmark ")) {
+                int taskNumber = Integer.parseInt(input.split(" ")[1]) - 1;
+                tasks[taskNumber].markAsNotDone();
+                printLine();
+                System.out.println("OK, I've marked this task as not done yet:");
+                System.out.println(tasks[taskNumber]);
+                printLine();
+            } else if (input.startsWith("todo ")) {
+                tasks[taskCount] = new Todo(input.substring(5)); // remove "todo " prefix
+                printLine();
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks[taskCount]);
+                taskCount++;
+                System.out.println("Now you have " + taskCount + (taskCount == 1 ? " task" : " tasks") + " in the list.");
+                printLine();
+            } else if (input.startsWith("deadline ")) {
+                String[] details = input.substring(9).split(" /by ");
+                tasks[taskCount] = new Deadline(details[0], details[1]);
+                printLine();
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks[taskCount]);
+                taskCount++;
+                System.out.println("Now you have " + taskCount + (taskCount == 1 ? " task" : " tasks") + " in the list.");
+                printLine();
+            } else if (input.startsWith("event ")) {
+                String[] details = input.substring(6).split(" /from | /to ");
+                tasks[taskCount] = new Event(details[0], details[1], details[2]);
+                printLine();
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks[taskCount]);
+                taskCount++;
+                System.out.println("Now you have " + taskCount + (taskCount == 1 ? " task" : " tasks") + " in the list.");
+                printLine();
+            } else {
+                printLine();
+                System.out.println("I'm sorry, I don't understand that command.");
+                printLine();
+            }
```

Comment:
Should this big if-else chain be extracted out? Any reason why (almost all of) the program is all in main instead of in its own functions?